mlflow/utils/validation.py

```diff
@@ -345,6 +345,19 @@ def _validate_experiment_name(experiment_name):
         )
 
 
+def _validate_start_run_experiment_id(experiment_id):
```

Comment:
```suggestion
def _validate_experiment_id(experiment_id):
```

Can we rename this to `_validate_experiment_id`?

Reply:
Left a brief explanation here:
https://github.com/mlflow/mlflow/pull/5720#discussion_r852537639
Do you think this logic should be combined?

Reply:
Thanks for the explanation! How about `_validate_experiment_id_type`?

Reply:
+1 I like it! :)